.github/workflows/ci.yml

```diff
@@ -91,4 +103,4 @@ jobs:
           invoke deps --gx-install -m '${{ matrix.markers }}' -r test
 
       - name: Run the tests
-        run: pytest -m "${{ matrix.markers }} and not unit" -vv -rEf
+        run: invoke ci-tests '${{ matrix.markers }} and not unit ' --up-services --verbose
```

Comment:
It looks like we should handle the suffix `and not unit` when we parse in the markers. See my comment below.